sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/CrashesListener.java

```diff
@@ -28,6 +28,7 @@
      * @return <code>true</code> if it requires to be confirmed by a user, otherwise <code>false</code>.
      * If this method returns <code>true</code>, {@link Crashes#notifyUserConfirmation(int)} must be called by yourself.
      */
+    @SuppressWarnings({"JavadocReference", "RedundantSuppression"})
```

Comment:
The javadoc links actually work but batch analysis complains. Then in file edit analysis disagrees so we need the second suppress to reconcile them...